core/lib/entity-classification.js

```diff
@@ -0,0 +1,52 @@
+import {Audit} from '../audits/audit.js';
+import UrlUtils from './url-utils.js';
+import {EntityClassification as ComputedEntityClassification} from '../computed/entity-classification.js';
+
+/**
+ * @param {LH.Artifacts} artifacts
+ * @param {LH.Audit.Context} context
+ */
+async function getEntityClassification(artifacts, context) {
```

Comment:
This function is only used in `runner.js` from what I can tell. Are you planning on using it elsewhere in future patches?

Reply:
Not that I can think of.

Reply:
If it's only used in `runner.js` then I think it should live there. I don't think a separate file is necessary.

Reply:
I'm generally in favor of breaking down long running code into separate functions/files or libs.

Why would we treat this differently than [StackPacks](https://github.com/GoogleChrome/lighthouse/blob/250d4f68c3d7dc17e36d818bc73ff118feb235eb/core/runner.js#L113)?

Reply:
stack-packs.js is more library-like in its interface. It takes just a single artifact whereas this file currently takes all of LH artifacts and a context. It also used computed artifact stuff. It's very unlike most of the other files we keep in `lib` in that respect.

It doesn't rely on any helper functions either (there's just one function here). If there were tons of other functions private to the implementation, I'd be for splitting it out

Reply:
sg!